lib/main/localizations/app_localizations.dart

```diff
@@ -1259,6 +1259,12 @@ class AppLocalizations {
         name: 'name');
   }
 
+  String get name_to_be_displayed {
```

Comment:
Should be use `lowerCamelCase` style for name

Reply:
No, because it exists from the old code. Changing it is quite time consuming and it is not really needed at this time. We should only change the new ones.